core/domain/takeout_service.py

```diff
@@ -42,34 +75,25 @@ def export_data_for_user(user_id):
                                 model export policy>
         }
     """
-    models_to_export = [
```

Comment:
Keep this old code, which lists the models explicitly. Have the discovery happen only in the tests. (See later comment.)

Reply:
I believe I have resolved this, let me know if issues persist